SPECS/python-distlib/python-distlib.spec

```
@@ -0,0 +1,192 @@
+%global srcname distlib
+%bcond_without check
+Summary:        Low-level components of distutils2/packaging, augmented with higher-level APIs
+Name:           python-distlib
+Version:        0.3.6
+Release:        1%{?dist}
```

Forgot to increment the `Release` tag:
```suggestion
Release:        2%{?dist}
```